modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -171,6 +171,9 @@ export default {
 
   .apos-notification__label {
     letter-spacing: 0.75px;
+    overflow: hidden;
```

Comment:
@stuartromanek Instead of a JS function `truncate`, I suggest this. What do you think. It will be more generic for all kinds of notifications.

Reply:
And more consistent in its effects, vs. variable width font mysteries

Reply:
@falkodev this approach is fine, we're using it elsewhere